This fails to build with (excerpt):

```
/build/source/source/ogre/CarReflection.cpp:96:31: error: 'class Ogre::Camera' has no member named 'setPosition'; did you mean 'getRealPosition'?
   96 |                         mCam->setPosition(Vector3::ZERO);
      |                               ^~~~~~~~~~~
      |                               getRealPosition
```

The offending line in the upstream source has not been changed in 11 years. [The latest (13.5) OGRE doc](https://ogrecave.github.io/ogre/api/latest/class_ogre_1_1_camera.html#abb5bad7918a07b1e7408f183b177d1a0) says `Ogre::Camera::setPosition()` is deprecated now, but it seems to still be there... 

I'm building with OGRE 13 because [the StuntRally 2.7 documentation](https://stuntrally.tuxfamily.org/wiki/doku.php?id=compile) says it's what it's been tested with.

I'll keep investigating tomorrow

--- a/pkgs/games/stuntrally/default.nix
+++ b/pkgs/games/stuntrally/default.nix
@@ -14,34 +14,50 @@
 , libXcursor
 , bullet
 , openal
+, tinyxml
+, tinyxml-2
 }:
 
+let
+  stuntrally_ogre = ogre.overrideAttrs (old: {
+    cmakeFlags = old.cmakeFlags ++ [
+      "-DOGRE_NODELESS_POSITIONING=ON"
+      "-DOGRE_RESOURCEMANAGER_STRICT=0"
+    ];
+  });
+  stuntrally_mygui = mygui.override {
+    withOgre = true;
+    inherit ogre;
+  };
+in
+
 stdenv.mkDerivation rec {
-  pname = "stunt-rally";
-  version = "2.6.2";
+  pname = "stuntrally";
+  version = "2.7";
 
   src = fetchFromGitHub {
     owner = "stuntrally";
     repo = "stuntrally";
     rev = version;
-    hash = "sha256-9I6hXsosqx+yYiEOEnPXQJHZkGtSU+JqThorwjemlc0=";
+    hash = "sha256-0Eh9ilIHSh/Uz8TuPnXxLQfy7KF7qqNXUgBXQUCz9ys=";
   };
   tracks = fetchFromGitHub {
     owner = "stuntrally";
     repo = "tracks";
     rev = version;
-    hash = "sha256-eZJAvkKe3PrXDzxTa5WFBHfltB3jhQh8puzOFDO9lso=";
+    hash = "sha256-fglm1FetFGHM/qGTtpxDb8+k2iAREn5DQR5GPujuLms=";
   };
 
   preConfigure = ''
-    ln -s ${tracks} data/tracks
+    rmdir data/tracks
+    ln -s ${tracks}/ data/tracks
   '';
 
   nativeBuildInputs = [ cmake pkg-config makeWrapper ];
   buildInputs = [
     boost
-    ogre
-    mygui
+    stuntrally_ogre
+    stuntrally_mygui
     ois
     SDL2
     libvorbis
@@ -49,6 +65,8 @@ stdenv.mkDerivation rec {
     libXcursor
     bullet
     openal
+    tinyxml
+    tinyxml-2
   ];
 
   meta = with lib; {

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -35269,7 +35269,7 @@ with pkgs;
   stt = callPackage ../tools/audio/stt { };
 
   stuntrally = callPackage ../games/stuntrally
-    { ogre = ogre1_9; mygui = mygui.override { withOgre = true; }; };
+    { };
 
   superTux = callPackage ../games/supertux { };
 

